### DISCUSSION TITLE
SQL Editor Raise Warning Log Location

### DISCUSSION THREAD

**Discussion by David-Rodriguez-Barrios** [supabase/supabase#22288] [Questions] [answered]
Hi, where can I find the Postgres Raise Notice and Warning Log location for SQL editor? I was creating PostgreSQL functions and was trying to use it to debug. I couldn't find it, so I had to use Raise Exception for it to appear in results.... How can I search for logs based on queries?

**Comment by Ziinc** [comment, marked as answer]
You should be able to find it under the [Postgres Logs section](https://supabase.com/dashboard/project/_/logs/postgres-logs)
